pkg/omf/functions/compat/refresh.fish

```diff
@@ -0,0 +1,14 @@
+function refresh -d "(deprecated) Refresh fish session by replacing current process"
+  printf 'warning: function %srefresh%s is deprecated and will be removed soon.\n %s\n' \
+  (set_color -u) (set_color normal) \
+  (status -t)[5] >&2
+
+  set -q CI
+    and return 0
+
+  type -q omf.core.reload
+    and omf.core.reload
+
```

Comment:
A comment here that `omf.core.reload` calls `exec` and that the lines below are only fallback will be helpful IMO.